app/controllers/ClaimPeriodStartController.scala

```diff
@@ -36,11 +37,10 @@ class ClaimPeriodStartController @Inject()(
   navigator: Navigator,
   getSession: GetSessionAction,
   getData: DataRetrievalAction,
-  requireData: DataRequiredAction,
   formProvider: ClaimPeriodStartFormProvider,
```

Comment:
`ClaimPeriodStartController` should be `ClaimPeriodController`

Reply:
ah , missed these, but just renamed all classes and changes pushed